demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -804,7 +804,7 @@ def test_GR107_IsDeprecatedContentItemInUsageValidatorListFiles_valid_playbook(
         pack_objects
     )
 
-    assert len(validation_results) == 0
+    assert len(validation_results) == 1
```

Comment:
Let's make sure we have all the cases we talked about to ensure we're covered now, and fore the future:
a deprecated and being collected -> b is using a
a deprecated and being collected -> b is deprecated using a
a not deprecated being collected -> b is used by a, b is deprecated
a not deprecated being collected -> b is used by a, b is not deprecated
a is being deprecated -> b uses a, b is using c, c is deprecated.
a is being not deprecated -> b uses a, b is using c, c is deprecated.

Reply:
Done.